src/server/middleware/sendHtml.js

```diff
@@ -69,30 +81,36 @@ export function renderStaticErrorPage(res) {
     message = 'Sorry, we are unable to load this page at this time.';
```

Comment:
is it possible to have this custom too? especially with internationalization?

Reply:
Possibly in another PR, this PR is just to allow a custom error page.

Reply:
this is meant to be the final catch all. we can not add anything that could risk an error being thrown here.

Reply:
Ideally if they wanted a custom message they would just use a custom error page right?